src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -98,7 +98,11 @@ static void handle_signal(int signum)
 {
   assert(signum == SIGINT || signum == SIGTERM);
   derr << "*** Got signal " << sig_str(signum) << " ***" << dendl;
-  ioctl(nbd, NBD_DISCONNECT);
+  dout(20) << __func__ << ": " << "sending NBD_DISCONNECT" << dendl;
+  if (ioctl(nbd, NBD_DISCONNECT) < 0)
+    cerr << "rbd-nbd: the device is not used" << std::endl;  
```

Comment:
It is not necessary "not used" in this context. I suggest:

   "rbd-nbd:: disconnect failed" << cpp_error(errno)

And output to derr, not cerr.

Also, I suggest always using curly brackets ({}), even for one line blocks.